ui/components/app/wallet-overview/eth-overview.test.js

```diff
@@ -381,6 +401,35 @@ describe('EthOverview', () => {
     });
   });
 
+  it('sends an event when clicking the Buy button: %s', () => {
+    const mockTrackEvent = jest.fn();
+
+    const mockedStore = configureMockStore([thunk])(mockStore);
+    const { queryByTestId } = renderWithProvider(
+      <MetaMetricsContext.Provider value={mockTrackEvent}>
+        <EthOverview />
+      </MetaMetricsContext.Provider>,
+      mockedStore,
+    );
+
+    const buyButton = queryByTestId(ETH_OVERVIEW_BUY);
+    expect(buyButton).toBeInTheDocument();
+    expect(buyButton).not.toBeDisabled();
+    fireEvent.click(buyButton);
+
+    expect(mockTrackEvent).toHaveBeenCalledWith({
+      event: MetaMetricsEventName.NavBuyButtonClicked,
+      category: MetaMetricsEventCategory.Navigation,
+      properties: {
+        account_type: mockEvmAccount1.type,
+        chain_id: CHAIN_IDS.MAINNET,
+        location: 'Home',
+        text: 'Buy',
+        token_symbol: expect.any(Object), // FIXME: Is this wrong?
```

Comment:
Should it be fixed?

Reply:
I've added a better comment rather than a FIXME. I might have put a FIXME to get back to this later and double-check so... Thanks for checking 🙈